### PR TITLE
Feature/standardise logging

### PR DESCRIPTION
**Pull request by eedwards0**
### Standardise logging
- Standardise logging output to date
- Include info level log statements for each post and get request in handlers
- Update logging to load the psc-verification-web namespace
- Update existing log statements to include the transaction and submission ID where appropriate

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/lib/logger.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { createLogger } from "@companieshouse/structured-logging-node";
 import ApplicationLogger from "@companieshouse/structured-logging-node/lib/ApplicationLogger";
+import { PSC_VERIFICATION_WEB_NAMESPACE } from "./constants";
+import { env } from "../config";
 
-export const logger: ApplicationLogger = createLogger(process.env.APP_NAME ?? "");
+export const logger: ApplicationLogger = createLogger(env.APP_NAME ?? PSC_VERIFICATION_WEB_NAMESPACE);
 
 // tslint:disable-next-line:no-console
 console.log(`env.LOG_LEVEL set to ${process.env.LOG_LEVEL}`);
```
Comment (PhilipGlover) [inline src/lib/logger.ts:9]: I suppose just to match, the `process` can be dropped from this console log too, to leave `${env.LOG_LEVEL}`

**Comment by PhilipGlover** [comment]
I'll approve next, but going to be cheeky and mention `src/routers/utils/index.ts`.
I see you've done `src/utils/date.ts` so does the above need to be included too?

**Review by PhilipGlover — APPROVED** [review]
- looks very good
- have run the code and seen the logging throughout the Individual journey
- my 2 comments are small and only mentioned for consideration

File: src/lib/logger.ts (hunk 1 above)
Comment (eedwards0) [reply src/lib/logger.ts:9]: Good spot - thanks!

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Pull request merged**